app/src/main/java/org/simple/clinic/addidtopatient/searchresults/AddIdToPatientSearchResultsScreenKey.kt

```diff
@@ -3,17 +3,24 @@ package org.simple.clinic.addidtopatient.searchresults
 import kotlinx.android.parcel.IgnoredOnParcel
 import kotlinx.android.parcel.Parcelize
 import org.simple.clinic.R
+import org.simple.clinic.patient.PatientSearchCriteria
 import org.simple.clinic.patient.businessid.Identifier
 import org.simple.clinic.router.screen.FullScreenKey
 
 @Parcelize
 data class AddIdToPatientSearchResultsScreenKey(
-    val fullName: String,
+    val criteria: PatientSearchCriteria,
     val identifier: Identifier
 ) : FullScreenKey {
 
   @IgnoredOnParcel
   override val analyticsName = "Add ID to Patient: Search Results"
 
   override fun layoutRes() = R.layout.screen_addidtopatientsearchresults
+
+  val fullName: String
```

Comment:
PTAL, maybe this should go.

Reply:
This was a temporary variable added during refactoring; I must've forgotten to remove it. Will remove.